configs/etl/reference.yaml

```diff
@@ -93,3 +93,15 @@ gwas_ingest:
     ingest_unpublished_studies: False
     p_value_cutoff: 5e-8
   machine: ${machine.default}
+
+tissue_enrichment:
+  inputs:
+    tissue_annotations: gs://genetics-portal-dev-analysis/xg1/EPIMAP/CHEERS_annotations/Full_epimap.parquet
+    summary_stats_credible_set: gs://genetics-portal-dev-analysis/xg1/CHEERS_credible_set/FM_220401_hg19.parquet
+    pics_credible_set: gs://genetics-portal-dev-analysis/xg1/CHEERS_credible_set/Non_sumstat_hg19.parquet
+  outputs:
+    summary_stats_enrichment: gs://genetics-portal-dev-analysis/xg1/CHEERS_enrichments/sumstats_enrichments/results.parquet
+    pics_enrichment: gs://genetics-portal-dev-analysis/xg1/pics_enrichments/results.parquet
```

Comment:
For the outputs, I would recommend to use the `${etl.outputs}/` root for placing files.